openmc/stats/multivariate.py

```diff
@@ -613,6 +616,117 @@ def from_xml_element(cls, elem):
         origin = [float(x) for x in elem.get('origin').split()]
         return cls(r, phi, z, origin=origin)
 
+class MeshIndependent(Spatial):
+    """Spatial distribution for a mesh.
+
+    This distribution allows one to specify a mesh to sample over and the scheme to sample over the entire mesh. 
+
+    .. versionadded:: 0.13
+
+    Parameters
+    ----------
+    mesh : openmc.MeshBase
+        The mesh instance used for sampling, mesh is written into settings.xml, mesh.id is written into the source distribution
+    strengths : Iterable of Real, optional
+        A list of values which represent the weights of each element
+
+
+    Attributes
+    ----------
+    mesh : openmc.MeshBase
+        The mesh instance used for sampling, mesh is written into settings.xml, mesh.id is written into the source distribution
+    strengths : Iterable of Real, optional
+        A list of values which represent the weights of each element
+
+    """
+
+    def __init__(self, mesh, strengths=None, volume_normalized=False):
+        self.mesh = mesh        
+        self.strengths = strengths
+        self.volume_normalized = volume_normalized
```

Comment:
Need a docstring for `volume_normalized`.

Reply:
Also, this is a misnomer now that I think about it. `volume_weighted` is probably more accurate. What do you think @NybergWISC?

Reply:
I think we were initially trying to avoid using "weights" because that is normally related to the particle instead of the source distributions. I think normalized works in this case, shows whether the volume of the tet is taken into account